<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainMenuUi</class>
 <widget class="QWidget" name="MainMenuUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>651</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="MainLayout" stretch="0,1,0,0,0,0,2,0">
    <property name="leftMargin">
     <number>25</number>
    </property>
    <property name="topMargin">
     <number>25</number>
    </property>
    <property name="rightMargin">
     <number>25</number>
    </property>
    <property name="bottomMargin">
     <number>25</number>
    </property>
    <item>
     <widget class="QLabel" name="SMELogoLarge">
      <property name="text">
       <string>{BIG_AF_LOGO}</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="MainLayoutSpacer1">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="NewProjectButton">
      <property name="text">
       <string>New Project</string>
      </property>
      <property name="shortcut">
       <string>Ctrl+N</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="OpenProjectButton">
      <property name="text">
       <string>Open Project</string>
      </property>
      <property name="shortcut">
       <string>Ctrl+O</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="EditGameFilesLayout">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="EditGameFilesButton">
        <property name="text">
         <string>Edit Game Files</string>
        </property>
        <property name="shortcut">
         <string>Ctrl+E</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="EditGameDirectoryToolButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>â–¼</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWidget" name="GameDirectoryWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>351</width>
         <height>44</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="GameDirectoryLayout">
        <property name="spacing">
         <number>4</number>
        </property>
        <item>
         <widget class="QLabel" name="TitleLabel">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Game directory:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="GameDirectoryLineEdit">
          <property name="text">
           <string>C:\Program Files (x86)\Steam\steamapps\common\Scrap Mechanic</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <spacer name="MainLayoutSpacer2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="CreatorLabel">
      <property name="text">
       <string>by Dicfact Soft</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
